### PR TITLE
Add import

### PR DESCRIPTION
**Pull request by KedziaPawel**

### CODE REVIEW

File: class/authenticator.ts [hunk 1]
```diff
@@ -42,15 +42,24 @@ export class Authenticator implements IAuthenticator {
...
-  async init(entropy: string) {
-    const buffer = Buffer.from(entropy, ENCODING);
+  async init({ entropy, mnemonic }: { entropy?: string; mnemonic?: string }) {
+    if (entropy === undefined && mnemonic === undefined) {
+      throw new Error('Not provided entropy or mnemonic');
+    }
+    const _entropy = mnemonic === undefined ? entropy : mnemonicToEntropy(mnemonic).toString(ENCODING);
+
+    if (_entropy === undefined) {
+      throw new Error('Couldn`t get entropy');
```
Comment (Kamill90) [inline class/authenticator.ts:52]: if those errors are visible for users we should keep them in locale files

File: src/screens/ImportAuthenticator/ImportAuthenticatorScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+interface State {
+  name: string;
+  mnemonic: string;
+  nameError: string;
+  mnemonicError: string;
+}
+
+type DynamicState = Pick<State, keyof State>;
+
+const MNEMONIC_LENGTH = 12;
```
Comment (Kamill90) [inline src/screens/ImportAuthenticator/ImportAuthenticatorScreen.tsx:29]: please let's move to consts

File: class/authenticator.ts (hunk 1 above)
Comment (KedziaPawel) [reply class/authenticator.ts:52]: there are not visible for user

File: src/screens/ImportAuthenticator/ImportAuthenticatorScreen.tsx (hunk 2 above)
Comment (KedziaPawel) [reply src/screens/ImportAuthenticator/ImportAuthenticatorScreen.tsx:29]: ok will do in next PR

**Pull request merged**